src/collectors/colmenaConfigurations.nix

```
@@ -24,24 +26,22 @@
     }
   ];
 
-  tranformToNixosConfig' = name: evaled: locatedConfig: let
+  transformToNixosConfigurations = name: evaled: locatedConfig: let
     config = {
       imports = [locatedConfig] ++ colmenaModules;
       _module.args = {inherit name;};
     };
   in
-    tranformToNixosConfig evaled config;
+    transformers.nixosConfigurations {inherit evaled config;};
```

```suggestion
    transformers.nixosConfigurations {inherit evaled locatedConfig;};
```
Or am I missing something?

---

Oh yes, you are right!